src/style/document-header.less

```diff
@@ -59,13 +59,26 @@
 .document-header {
     height: @toolbar-button-size;
     line-height: @toolbar-button-size;
-    max-width: 100%;
-    width: 100%;
+    max-width: 93%;
+    width: 93%;
     position: relative;
     overflow: hidden;
     display: flex;
 }
 
+.generator-header {
+    height: @toolbar-button-size;
+    line-height: @toolbar-button-size;
+    width: 7%;
+    max-width: 7%;
```

Comment:
I think this needs a fixed width. If the screen is very wide, the icon drifts away from the right edge:
![image](https://cloud.githubusercontent.com/assets/1075154/9864380/d62585e4-5afd-11e5-815f-a18d70d747fc.png)